compiler/rustc_interface/src/passes.rs

```diff
@@ -954,6 +954,7 @@ fn analysis(tcx: TyCtxt<'_>, (): ()) -> Result<()> {
             if !tcx.sess.opts.debugging_opts.thir_unsafeck {
                 rustc_mir_transform::check_unsafety::check_unsafety(tcx, def_id);
             }
+            tcx.ensure().has_ffi_unwind_calls(def_id);
```

Comment:
This seems to be called twice: here and in `mir_const`. Is this intentional?

Reply:
Yes. This query uses `mir_built`, which will be stolen later. So I followed the same pattern as `check_unsafety` (which also uses `mir_built`, by making sure it's evaluated before a query that could steal it.